screen-orientation/non-fully-active.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<p id="#fragment"></p>
+<script>
+  async function attachIFrame() {
+    const iframe = document.createElement("iframe");
+    document.body.appendChild(iframe);
+    await new Promise((resolve) => {
+      iframe.onload = resolve;
+      iframe.src = "resources/empty.html";
```

Comment:
Any reason we can't use about:blank?

Reply:
Was unsure about same origin check. Can use about:blank if it's treated as same origin.